tests/pages/api/chooseValidity.test.ts

```diff
@@ -30,16 +35,18 @@ describe('chooseValidity', () => {
     });
 
     it('should set the validity stages session according to the specified number of fare stages', () => {
-        const { req, res } = getMockRequestAndResponse({ cookieValues: {}, body: { validityInput: '6' } });
-
-        const mockSetSession = jest.fn();
-
-        (updateSessionAttribute as {}) = jest.fn().mockImplementation(() => {
-            mockSetSession();
+        const { req, res } = getMockRequestAndResponse({
+            cookieValues: {},
+            body: { validityInput: '6', duration: 'day' },
         });
 
         chooseValidity(req, res);
 
-        expect(mockSetSession).toBeCalledTimes(1);
+        expect(updateSessionAttributeSpy).toBeCalledTimes(1);
```

Comment:
Is this `expect` not covered by the below?

Reply:
No, as it could potentially be called multiple times, and the below just checks if it was ever called with those things.